cli/commands/migrate.ts

```diff
@@ -169,7 +169,15 @@ export const migrate = async (
   logger.info(`Sent ${nTx} transaction${nTx === 1 ? '' : 's'} & spent ${EtherSymbol} ${spent}`)
 
   if (chainId == 1337) {
-    await (cli.wallet.provider as providers.JsonRpcProvider).send('evm_setAutomine', [autoMine])
+    await setAutoMine(cli.wallet.provider as providers.JsonRpcProvider, autoMine)
```

Comment:
Oooh now I remember what we were trying to do with the automine settings here. The idea was to set it to true above (before starting the migration), and then this step at the end would set it back to false when we're done, because at the time we were disabling automine by default when running a local hardhat node.

Now that we have the autoMine task flag, what this is actually doing is setting up the autoMine for _after_ the migration - the migration itself is always run with automine = true.

So I wonder if the best thing to do here would be to simply restore whatever automine setting was set before we start the migration?

Reply:
Yes I think I agree that's the most sensible thing to do, I'll do that! Do you know why the hardhat network sets autoMine to false in the network config?